src/components/modal.tsx

```diff
@@ -21,29 +22,36 @@ export interface MotionStyle {
 }
 
 export class Modal extends React.Component<ModalProps, ModalState> {
+    static _willEnter(): MotionStyle {
+        return { opacity: 0 };
+    }
+
+    static _willLeave(): MotionStyle {
```

Comment:
I don't think these methods should be static, even though that's theoretically possible. They are part of the class.

Reply:
Again this is one of the rules (https://eslint.org/docs/rules/class-methods-use-this). If a function does not use `this` then it should be static. Not my favourite rule though